smu/parser/smu_utils_lib_test.py

```diff
@@ -782,31 +783,31 @@ def setUp(self):
     self.conformer = dataset_pb2.Conformer()
     properties = self.conformer.properties
     # A STANDARD field
-    properties.single_point_energy_pbe0d3_6_311gd.value = 1.23
+    properties.initial_geometry_energy.value = 1.23
     # A COMPLETE field
-    properties.homo_pbe0_aug_pc_1.value = 1.23
+    properties.zpe_unscaled.value = 1.23
     # An INTERNAL_ONLY field
-    properties.nuclear_repulsion_energy.value = 1.23
+    properties.compute_cluster_info = "some value"
```

Comment:
Quotation marks don't match those used everywhere else.
Rather than 'some value', use 'unused', or 'not set', or simply an empty string.